mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/MqttSubscriber.java

```diff
@@ -0,0 +1,22 @@
+package com.cumulocity.mqtt.connect.client;
+
+/**
+ * Interface for subscribing to messages from the server.
+ * <p>
+ * {@link MqttSubscriber} instances are setup using {@link MqttClient#buildSubscriber(MqttConfig)}
+ */
+public interface MqttSubscriber extends AutoCloseable {
+
+    /**
+     * Sets a {@link MqttMessageListener} for the subscriber
```

Comment:
suggestion: Does this actually start the subscription and show we say that in the comment? For example; 'Starts this subscriber, using the parameter MqttMessageListener to handle its received messages'. Maybe a better wording than that even.

Reply:
Changes, thanks!